src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1224,13 +1221,10 @@
 (schema/defn ^:always-validate get-custom-oid-mappings :- (schema/maybe OIDMappings)
   "Given a path to a custom OID mappings file, return a map of all oids to
   shortnames"
-  [custom-oid-mapping-file :- (schema/maybe schema/Str)]
-  (when custom-oid-mapping-file
-    (if (fs/file? custom-oid-mapping-file)
-      (let [oid-mappings (:oid_mapping (yaml/parse-string (slurp custom-oid-mapping-file)))]
-        (into {} (for [[oid names] oid-mappings] [(name oid) (keyword (:shortname names))])))
-      (log/error (format "%s is not a valid file path. Custom OID Mappings will not be loaded."
-                         custom-oid-mapping-file)))))
+  [custom-oid-mapping-file :- schema/Str]
+  (when (fs/file? custom-oid-mapping-file)
+    (let [oid-mappings (:oid_mapping (yaml/parse-string (slurp custom-oid-mapping-file)))]
+      (into {} (for [[oid names] oid-mappings] [(name oid) (keyword (:shortname names))])))))
```

Comment:
I would probably change these two `when` usages to `if` and when `fs/file?` returns falsey log a message at DEBUG (probably?) like `No custom OID mapping configuration file found at $custom-oid-mapping-file, no custom OID mappings loaded.` or something like that (and similar for CSR attributes).